api/src/main/java/com/ichi2/anki/api/AddContentApi.java

```diff
@@ -393,7 +393,7 @@ public Long addNewBasic2Model(String name) {
 
     /**
      * Insert a new model into AnkiDroid.
-     * See the <a href="http://ankisrs.net/docs/manual.html#cards-and-templates">Anki Desktop Manual</a> for more help
+     * See the <a href="https://ankisrs.net/docs/manual.html#cards-and-templates">Anki Desktop Manual</a> for more help
```

Comment:
Verified correct / current link

```suggestion
     * See the <a href="https://docs.ankiweb.net/templates/intro.html">Anki Desktop Manual</a> for more help
```